Client/Frontend/Browser/SessionData.swift

```diff
@@ -6,6 +6,29 @@ import Foundation
 
 import Shared
 
+private func sanitize(urls: [URL]) -> [URL] {
```

Comment:
maybe should be called migration? I have it also healing malformed nested internal urls, I wonder if GCDWebServer was silently handling these. I need to investigate this further if I can repro this case again.